examples/Civis_WBCD_2017.ipynb

```diff
@@ -0,0 +1,676 @@
+    "import civis\n",
+    "from civis.ml import ModelPipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Pandas version: 0.19.2\n",
+      "Civis version: 1.5.2\n",
+      "Xgboost version: 0.6\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Pandas version: {}\".format(pd.__version__))\n",
+    "print(\"Civis version: {}\".format(civis.__version__))\n",
+    "print(\"Xgboost version: {}\".format(xgboost.__version__))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Downloading Data\n",
+    "- First things first, we need to grab the data from UCI ML Repository"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "breast_cancer_data.csv already downloaded\n",
+      "CPU times: user 508 µs, sys: 692 µs, total: 1.2 ms\n",
+      "Wall time: 767 µs\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "DATA_URL = \"http://mlr.cs.umass.edu/ml/machine-learning-databases/breast-cancer-wisconsin/breast-cancer-wisconsin.data\"\n",
+    "\n",
+    "if not os.path.isfile('breast_cancer_data.csv'):\n",
+    "    print(\"Fetching data...\")\n",
+    "    opener = urllib.request.URLopener()\n",
+    "    opener.retrieve(DATA_URL, \"breast_cancer_data.csv\")\n",
+    "    print(\"File Size: {:.1f} KB\".format(os.path.getsize('breast_cancer_data.csv') / (1024*100)))\n",
+    "    print(\"Finished\")\n",
+    "else:\n",
+    "    print(\"breast_cancer_data.csv already downloaded\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Munging Data\n",
+    "- We'll add in the column names, and tell pandas that null values are encoded as a '?'\n",
+    "- We can peek at shape. This is a small table, which I chose to allow for quick training.\n",
+    "- We'll recode the response 4 (cancer) and 2 (benign) as 1 and 0 respectively. **I don't have to do this**, but to me it feels more intuitive to deal with ones and zeros"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Shape:  (699, 11)\n"
+     ]
+    }
+   ],
+   "source": [
+    "COL_NAMES = ['sample_id', 'clump_thickness', 'uniformity_cell_size', 'uniformity_cell_shape',\n",
+    "            'marginal_adhesion', 'epithelial_cell_size', 'bare_nuclei', 'bland_chromatin',\n",
+    "            'normal_nucleoli', 'mitoses', 'is_cancer']\n",
+    "df = pd.read_csv(\"breast_cancer_data.csv\", names=COL_NAMES, na_values='?')\n",
+    "\n",
+    "print(\"Shape: \", df.shape)\n",
+    "\n",
+    "df['is_cancer'] = [1 if val == 4 else 0 for val in df['is_cancer'] ]\n",
+    "# df.head()"
```

Comment:
Was doing that on purpose, per @samcarlos feedback that it would be nice to have code to actually look at the data, even if it were commented out!